cpp/include/cuspatial/utility.hpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cusparse.h"
+
+namespace detail {
+
```

Comment:
This should all probably be rewritten to match CUDF and RMM (exceptions with CUSPATIAL_EXPECTS, CUSPARSE_TRY macros, etc.)

Not in this PR, but an issue should be filed.

Reply:
Done, thanks.